normandy/recipes/filters.py

```diff
@@ -353,6 +354,35 @@ def capabilities(self):
         return {"jexl.context.env.version", "jexl.transform.versionCompare"}
 
 
+class ProfileCreateDateFilter(BaseFilter):
+
+    type = "profile_creation_date"
+    direction = serializers.CharField()
+    date = serializers.DateField()
+
+    def to_jexl(self):
+        direction = self.initial_data["direction"]
+        date = self.initial_data["date"]
+
+        days = (datetime.strptime(date, "%Y-%M-%d") - datetime(1970, 1, 1)).days
+
+        if direction == "olderThan":
+            symbol = "<"
```

Comment:
I think it would be good if these were `<=`/`>` or `<`/`>=`, so that the two opposing filters meet up exactly. We often use this kind of filter in a pair of recipes that we expect to be complements.